File: 009/radians.c
#include "radians.h"

void goc_phan_tu(double a) {
	a = a/60;
	double radians_a = a * M_PI / 180;
	double cos_a = cos(radians_a);

	if ( 0 <= a && a < 90 ) {
		printf("x thuoc goc phan tu thu I\n");
	} else if ( 90 <= a && a < 180 ) {
		printf("x thuoc goc phan tu thu II\n");
	} else if ( 180 <= a && a < 270 ) {
		printf("x thuoc goc phan tu thu III\n");
	} else if ( 270 <= a && a <= 360 ) {
		printf("x thuoc goc phan tu thu IV\n");
	} else {
		printf("Goc khong hop le!\n");
	}

	printf("cos(x) = %lf\n", cos_a);
}

File: 009/radians.rb
def goc_phan_tu(a) 
  a = a/60
  radians_a = a * Math::PI / 180
  cos_a = Math.cos(radians_a)
  
  puts "x thuoc goc phan tu thu I" if 0 <= a && a < 90
  puts "x thuoc goc phan tu thu II" if 90 <= a && a < 180
  puts "x thuoc goc phan tu thu III" if 180 <= a && a < 270
  puts "x thuoc goc phan tu thu IV" if 270 <= a && a < 360
  
  puts "cos(x) = #{'%.6f' %cos_a}"

end
